core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -251,7 +251,9 @@ angular.module('oppia').directive('explorationEditorPage', [
             ]).then(function(combinedData) {
               var explorationData = combinedData[0];
               var featuresData = combinedData[1];
-
+              if (explorationData.context) {
```

Comment:
If you're only using the story ID as a boolean here, pass a boolean from the backend.

Reply:
Done